site/content/en/docs/drivers/_index.md

```diff
@@ -8,9 +8,12 @@ description: >
 aliases:
   - /docs/reference/drivers
 ---
-minikube can be deployed as a VM, a container, or bare-metal.
+Minikube can be deployed as a VM, a container, or bare-metal where a server can be used as a single tenant.
```

Comment:
For style consistency, use lowercase `M` for minikube.

Reply:
Sure that sounds great but that's actually the first letter so I kept it like that.

Reply:
I would also recommend using lowercase `m` here, even though it is the first letter.